libfuse/tlf.go

```diff
@@ -152,8 +152,31 @@ func (tlf *TLF) Attr(ctx context.Context, a *fuse.Attr) error {
 	return dir.Attr(ctx, a)
 }
 
+// tlfLoadavoidingLookupNames specifies a set of directory entry names
+// that should NOT cause a TLF to be fully loaded and identified on a
+// lookup.  If the directory is not yet loaded and one of these names
+// are looked-up, then ENOENT will be returned automatically.  This is
+// to avoid unnecessary loading and tracker popups when listing the
+// folder list directories.  For example, when macOS finder opens
+// `/keybase/private`, it looks up `.localized` in every TLF
+// subdirectory to see if it should translate the TLF folder name or
+// not, which can cause a tracker popup storm (see KBFS-2649).
+var tlfLoadavoidingLookupNames = map[string]bool{
```

Comment:
s/avoiding/Avoiding

Reply:
I did this on purpose because I wanted to evoke "load-avoiding lookup" rather than "load avoiding lookup" (which is ambiguous), and it seemed like the right way to do it.  But I guess I don't feel too strongly about.

Reply:
it's pretty hard to read though.